man/man8/zfs.8

```diff
@@ -699,6 +701,12 @@ The \fBuserused@\fR... properties are not displayed by \fBzfs get all\fR. The us
 .RE
 Files created on Linux always have POSIX owners.
 
+.RS 4n
+The \fBuserobjused\fR is similar to \fBuserused\fR but instead it counts the amount of object count consumed by \fIuser\fR.
+Due to the fact that zfs may use extra objects to store xattr, \fBuserobjused\fR may count more objects than the number
+from user's perspective.
```

Comment:
How about:

When the property \fBxattr=on\fR is set on a fileset, ZFS will create additional objects per-file to store extended attributes.  These additional objects are reflected in the \fBuserobjused\fR value and are counted against the user's \fBuserobjquota.\fR.  When a filesystem is configured to use \fBxattr=sa\fR no additional internal objects are required.